From c288b61fb149150b73cce955f2de4bb33a221df1 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 10 Oct 2017 19:54:24 +0000
Subject: [PATCH 1/2] sphinx-build isn't postfixed with 3

---
 docs/hawkey/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/hawkey/CMakeLists.txt b/docs/hawkey/CMakeLists.txt
index 321c220..417ac12 100644
--- a/docs/hawkey/CMakeLists.txt
+++ b/docs/hawkey/CMakeLists.txt
@@ -4,7 +4,7 @@
 if (${PYTHON_VERSION_MAJOR} STREQUAL "2")
     SET(SPHINX_BUILD_NAME "sphinx-build")
 else()
-    SET(SPHINX_BUILD_NAME "sphinx-build-3")
+    SET(SPHINX_BUILD_NAME "sphinx-build")
 endif()
 
 IF(WITH_HTML)
-- 
2.20.1

